docs/tutorials/porch-development-environment/README.md

```diff
@@ -0,0 +1,148 @@
+
+
+You can reach the Gitea web interface on the address reported by the following command:
+```
+kubectl get svc -n gitea gitea        
+NAME    TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)                       AGE
+gitea   LoadBalancer   10.197.10.118   172.18.255.200   22:31260/TCP,3000:31012/TCP   8m35s
+```
+
+# Install the Porch function runner
+
+The Porch server requires that the Porch function runner is executing. To install the Porch function runner on the Kind management cluster, execute the following commands.
+
+```
+kubectl apply -f https://raw.githubusercontent.com/nephio-project/catalog/main/nephio/core/porch/0-packagerevs.yaml
+kubectl apply -f https://raw.githubusercontent.com/nephio-project/catalog/main/nephio/core/porch/0-packagevariants.yaml
+kubectl apply -f https://raw.githubusercontent.com/nephio-project/catalog/main/nephio/core/porch/0-packagevariantsets.yaml
+kubectl apply -f https://raw.githubusercontent.com/nephio-project/catalog/main/nephio/core/porch/0-repositories.yaml
+kubectl apply -f https://raw.githubusercontent.com/nephio-project/catalog/main/nephio/core/porch/1-namespace.yaml
+kubectl apply -f https://raw.githubusercontent.com/nephio-project/catalog/main/nephio/core/porch/2-function-runner.yaml
+
+kubectl wait --namespace porch-system \
+                --for=condition=ready pod \
+                --selector=app=function-runner \
+                --timeout=300s
+```
+
+The Porch function runner should now be executing:
+
+```
+kubectl get pod -n porch-system --selector=app=function-runner
+NAME                              READY   STATUS    RESTARTS   AGE
+function-runner-67d4c7c7b-7wm97   1/1     Running   0          16m
+function-runner-67d4c7c7b-czvvq   1/1     Running   0          16m
+```
+
+Expose the `function-runner` service so that the Porch server running in Visual Studio Code can reach it, change the service type from `ClusterIP` to `LoadBalancer`:
+
+```
+kubectl edit svc -n porch-system function-runner
+
+31c31
+<   type: ClusterIP
+---
+>   type: LoadBalancer
+```
+
+Now check that the `function-runner` service has been assigned an IP address external to the cluster:
+```
+kubectl get svc -n porch-system function-runner
+NAME              TYPE           CLUSTER-IP       EXTERNAL-IP      PORT(S)          AGE
+function-runner   LoadBalancer   10.197.168.148   172.18.255.201   9445:31794/TCP   22m
+```
+
+# Install Porch resources for standalone execution
+
+The Porch server requires that the following resources are defined in the K8S cluster where it is executed:
+
+- The `porch-system` namespace, an API Service called `apiservice.apiregistration.k8s.io/v1alpha1.porch.kpt.dev` and the `service.api` service to expose the API Service. These resources are defined in the the file `deployments/local/localconfig.yaml`
+- the `repositories.config.porch.kpt.dev` and `functions.config.porch.kpt.dev` CRDs. These CRDs are defined in the `api/porchconfig/v1alpha1/` directory.
```

Comment:
Upper case "The"?
Same on list item below?

Reply:
Done, tidied up.